src/bindings/python/src/openvino/runtime/ie_api.py

```diff
@@ -515,6 +515,45 @@ def compile_model(
             super().compile_model(model, device_name, {} if config is None else config),
         )
 
+    def compile_model(
+        self,
+        model_buffer: str,
+        weight_buffer: bytes,
+        device_name: Optional[str] = None,
+        config: Optional[dict] = None,
+    ) -> CompiledModel:
```

Comment:
Could we add some test cases for this API?

Reply:
Add unit test